widgets/odm-related-content-widget.php

```diff
@@ -52,40 +52,43 @@ public function widget( $args, $instance ) {
     $limit = isset($instance['limit']) ? $instance['limit'] : -1;
     $type = isset($instance['type']) ? $instance['type'] : 'dataset';
 		$template = isset($instance['template']) ? $instance['template'] : 'default';
+		$related_content = get_post_meta($post->ID,'related_content',true);
+		$data = json_decode($related_content,true);
+		
+		if ($data !== null):
```

Comment:
Yes, just add condition. That should apply to all the widgets. As you see, I just added the condition to the ODC content the same category as well.